### PR TITLE
Fix: bad link & a few other doc issues

### PR DESCRIPTION
**Pull request by lwasser**
i did another pass on the docs and found a few bad links and missing nav bars. This just cleans that up.

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -75,6 +72,9 @@
 autodoc_pydantic_settings_show_json = False
 autosummary_generate = True
 
+# Suppress warnings for duplicate object description
+suppress_warnings = ["autodoc.*", "ref.ref", "ref.duplicate"]
+
```
Comment (lwasser) [inline docs/conf.py:77]:
```suggestion
```

File: docs/contributing/intro.md [hunk 2]
```diff
@@ -47,19 +45,6 @@ Learn about our build and release workflow. We use version control based version
 If you are new to contributing to open source software, these resources will help you get started.
 :::
 
-
```
Comment (lwasser) [inline docs/contributing/intro.md:50]: This is in the what's new section now so i removed it from contributing.

File: docs/whats-new/changelog.md [hunk 3]
```diff
@@ -1,10 +1 @@
-:::{toctree}
```
Comment (lwasser) [inline docs/whats-new/changelog.md:1]: Duplicate toctree call

File: docs/whats-new/stravalib-2.md [hunk 4]
```diff
@@ -1,5 +1,14 @@
 # Stravalib V2 Migration Guide
 
+:::{toctree}
```
Comment (lwasser) [inline docs/whats-new/stravalib-2.md:3]: Added the guide to the what's new section and also put the toctree here

File: docs/whats-new/stravalib-2.md [hunk 5]
```diff
@@ -1,5 +1,14 @@
...
+:hidden:
+:caption: What's New
+
+What's New <self>
+Changelog <changelog>
+blah </reference/index>
```
Comment (jsamoocha) [inline docs/whats-new/stravalib-2.md:9]: What about this?

**Review by jsamoocha — COMMENTED** [review]
Thanks! I may be mistaken, but it looks like one placeholder-like line should be removed. Besides that, this is good to merge.

File: docs/whats-new/stravalib-2.md (hunk 5 above)
Comment (lwasser) [reply docs/whats-new/stravalib-2.md:9]: 😆 @jsamoocha you're being nice. that was a mistake. the toctree wasn't populating and i was testing it by adding additional items to see if it was a changelog issue again. then i forgot to remove. good catch!

Comment (lwasser) [inline docs/whats-new/stravalib-2.md:9]:
```suggestion
```

**Comment by lwasser** [comment]
Yahoo!

**Pull request merged**